docs/tutorials/creating-an-ioc.rst

```diff
@@ -233,17 +133,5 @@ reading and writing the current value of the record.  For IN records calling
 (all IN records are by default created with ``SCAN='I/O Intr'``).
 
 
-Initialising the IOC
---------------------
-
-This is simply a matter of calling two functions:
-:func:`~softioc.builder.LoadDatabase` and :func:`~softioc.softioc.iocInit`,
-which must be called in this order.  After calling
-:func:`~softioc.builder.LoadDatabase` it is no longer possible to create PVs.
-
-It is sensible to start any server background activity after the IOC has been
-initialised by calling :func:`~softioc.softioc.iocInit`.  After this has been
-done (:class:`cothread.Spawn` is recommended for initiating persistent background
-activity) the top level script must pause, as as soon as it exits the IOC will
-exit.  Calling :func:`~softioc.softioc.interactive_ioc` is recommended for this
-as the last statement in the top level script.
+..  _cothread: https://github.com/dls-controls/cothread
```

Comment:
You shouldn't need these links as intersphinx should pick up `cothread` and `epicsdbbuilder`

Reply:
Resolved in commit a873d0e